### PR TITLE
sync readme

### PR DESCRIPTION
**Pull request by blagoev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,15 @@ This repository holds the source code for the Realm SDK for Flutter™ and Dart
 
 **This project is in the Beta stage. The API should be quite stable, but occasional breaking changes may be made.**
 
+## Features
+
+* **Mobile-first:** Realm is the first database built from the ground up to run directly inside phones, tablets, and wearables.
+* **Simple:** Data is directly [exposed as objects](https://docs.mongodb.com/realm/dotnet/objects/) and [queryable by code](https://docs.mongodb.com/realm/dotnet/query-engine/), removing the need for ORM's riddled with performance & maintenance issues. Plus, we've worked hard to [keep our API down to just a few common classes](https://pub.dev/documentation/realm/latest/): most of our users pick it up intuitively, getting simple apps up & running in minutes.
```
Comment (desistefanova) [inline README.md:15]:
```suggestion
* **Simple:** Data is directly [exposed as objects](https://www.mongodb.com/docs/realm/sdk/flutter/realm-database/define-realm-object-schema/) and [queryable by code](https://www.mongodb.com/docs/realm/sdk/flutter/realm-database/read-and-write-data/), removing the need for ORM's riddled with performance & maintenance issues. Plus, we've worked hard to [keep our API down to just a few common classes](https://pub.dev/documentation/realm/latest/): most of our users pick it up intuitively, getting simple apps up & running in minutes.
```

File: README.md [hunk 2]
```diff
@@ -9,6 +9,15 @@ This repository holds the source code for the Realm SDK for Flutter™ and Dart
...
+* **Modern:** Realm supports latest Dart and Flutter versions and is build with sound null-safety.
+* **Fast:** Realm is faster than even raw SQLite on common operations while maintaining an extremely rich feature set.
+* **[Device Sync](https://www.mongodb.com/atlas/app-services/device-sync)**: Makes it simple to keep data in sync across users, devices, and your backend in real-time. [Get started](http://mongodb.com/realm/register?utm_medium=github_atlas_CTA&utm_source=realm_dart_github) for free with a template application that includes a cloud backend and Sync.
+
+
```
Comment (desistefanova) [inline README.md:20]: Maybe it is good to mention the new link to Get Started page for flexible Sync https://www.mongodb.com/docs/atlas/app-services/get-started/

**Review by desistefanova — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (blagoev) [reply README.md:15]: that url is not the same so I suggest we keep it to .NET until we have the correct section in the Flutter docs https://www.mongodb.com/docs/realm/sdk/flutter/realm-database/define-realm-object-schema/

File: README.md (hunk 2 above)
Comment (blagoev) [reply README.md:20]: We should be in sync with the rest of the SDKS

File: README.md (hunk 1 above)
Comment (desistefanova) [reply README.md:15]: I think these two links have the same meaning. https://www.mongodb.com/docs/realm/sdk/flutter/realm-database/define-realm-object-schema/ and https://www.mongodb.com/docs/realm/sdk/flutter/realm-database/read-and-write-data/
Do we expect to have the same pages as .NEt SDK?

Comment (blagoev) [reply README.md:15]: They do not have similar meaning. The contents is very different. And yes we should have these fundamental pages documented for Flutter as well.

Comment (desistefanova) [reply README.md:15]: OK

**Review by desistefanova — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (nirinchev) [reply README.md:15]: I think we can as well remove the links and leave it as text. I think linking to the .NET docs here will be more confusing than helpful.

**Pull request merged**